mypy/fastparse.py

```diff
@@ -651,7 +651,7 @@ def visit_Import(self, n: ast3.Import) -> Import:
     def visit_ImportFrom(self, n: ast3.ImportFrom) -> ImportBase:
         assert n.level is not None
         if len(n.names) == 1 and n.names[0].name == '*':
-            assert n.module is not None
+            assert n.module is not ''
```

Comment:
This is actually not what I meant. The fix should at least replace ``None`` with ``''``.

Reply:
```
if n.module is None:
    n.module = ''
```

Is this what you mean?

Reply:
Yes, but it may be more subtle than that. ``--strict-optional`` handling requires care, so I need to investigate this myself in any case.

Reply:
OK, it looks like you can write
```python
        mod = n.module if n.module is not None else ''
        i = ImportAll(mod, n.level)
```
Also please make the corresponding change in ``fastparse2.py`` and add tests (check that the import does not crash in both Python 2 and Python mode).

(I think this is something we should get into the next release, so it is better not to wait with this.)

Reply:
@matzipan Is there any progress on this? It is a crash so I don't want the fix to wait indefinitely.